invenio_accounts/views.py

```diff
@@ -28,9 +28,25 @@
 
 from flask import Blueprint
 
+from flask_menu import register_menu
+
 blueprint = Blueprint(
     'invenio_accounts',
     __name__,
     template_folder='templates',
     static_folder='static',
 )
+
+
+# Register Menus
+@register_menu(blueprint, 'settings.account', 'Account', active_when=lambda: False)
+def menu_header():
+    """Register account menu header."""
+    pass
+
+
+@blueprint.route('/change')
```

Comment:
The URL is determined by Flask-Security configuration value so we can't hardcode it.